.github/actions/conventional-checks/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Conventional Checks
+
+description: Check if the branch name and pull request title follow conventional checks.
+
+runs:
+  using: composite
+  steps:
+    - name: Get branch name (merge)
+      if: github.event_name != 'pull_request'
+      shell: bash
+      run: echo "BRANCH_NAME=$(echo ${GITHUB_REF#refs/heads/})" >> $GITHUB_ENV
+
+    - name: Get branch name (pull request)
+      if: github.event_name == 'pull_request'
+      shell: bash
+      run: echo "BRANCH_NAME=$(echo ${GITHUB_HEAD_REF})" >> $GITHUB_ENV
+
+    - name: Check branch name
+      shell: bash
+      run: |
+        local_branch="${{ env.BRANCH_NAME }}"
+
+        valid_branch_regex="^(improvement|bug|feature|test|tmp)\/AAE-[0-9]+-[A-Za-z0-9._-]+$"
```

Comment:
we need 2 inputs for making this action actually reusable by everyone else here on build-tools

Reply:
I agree the regexp should be an input, now to ease up our usage maybe we can use this one as a default value (?) cc @andrea-ligios since you did not want the regexp to be duplicated in caller workflows

Reply:
maybe we can also extract the jira project name, so that the regexp is a valid default value, without hardcoding the jira ref at all

Reply:
Yes please... let's standardize branch names and PR names leaving out only the project... cross-bdu consistency is not required but can't be bad, right?

Reply:
Done

Reply:
Late thought: on some repos we accept different jira projects, do we want to cover that use case?

Reply:
I thought about it as well but in the end it should work with jira_project_id defined as `(PR1|PR2)`?

Reply:
Right, if this is confirmed, would be nice to document it then